src/pages/Scholarships.jsx

```diff
@@ -50,6 +50,23 @@ function ScholarshipsPage() {
     history.replace({ search: queryString.stringify(params) });
   };
 
+  const sortBy = params.sortBy ?? DEFAULT_SORT_BY;
+
+  if (
+    sortBy !== 'deadlineAsc' &&
+    sortBy !== 'deadlineDesc' &&
+    sortBy !== 'amountAsc' &&
+    sortBy !== 'amountDesc'
+  ) {
```

Comment:
I suggest using [`hasOwnProperty`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty) here.

Reply:
Look at comment below and see if that's what you mean